Makefile

```diff
@@ -41,6 +41,10 @@ test: build
 		--entrypoint=python \
 		texastribune/checkout:dev /usr/local/bin/py.test /app/tests.py --cov=/app
 
+test-new:
+	docker-compose up -d --build
+	docker-compose run --entrypoint=python test -m pytest "tests.py" --cov=.
```

Comment:
**suggestion (non-blocking, dx):**  Small suggestion to move the entrypoint & command for running the test container in the `docker-compose.yml` (see below).  Then you could say `docker-compose run test` and it would do what you have here.  

```suggestion
	docker-compose run test
```
That would mean if you wanted to, say, start a bash container for the image, you could do `docker-compose run --entrypoint=bash test`.